nvidia_deeplearningexamples_gpunet.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+title: GPUNet
+summary: GPUNet is a new family of Convolutional Neural Networks designed to max out the performance of NVIDIA GPU and TensorRT.
+category: researchers
+image: nvidia_logo.png
+author: NVIDIA
+tags: [vision]
+github-link: https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/Classification/GPUNet
+github-id: NVIDIA/DeepLearningExamples
+featured_image_1: classification.jpg
+featured_image_2: no-image
+accelerator: cuda
+---
+
+
+### Model Description
+GPUNets are a new family of deployment and production ready Convolutional Neural Networks from NVIDIA auto-designed to max out the performance of NVIDIA GPU and TensorRT. 
+
+Crafted by NVIDIA AI using novel Neural Architecture Search(NAS) methods, GPUNet demonstrates state-of-the-art inference performance up to 2x faster than EfficientNet-X and FBNet-V3. This notebook allows you to load and test all the the GPUNet model implementation listed in our [CVPR-2022 paper](https://arxiv.org/pdf/2205.00841.pdf). You can use this notebook to quickly load each one of listed models to perform inference runs.
+
+### Example
+In the example below the pretrained ***GPUNet-0*** model is loaded by default to perform inference on image and present the result. You can switch the default pre-trained model loading from GPUNet-0 to one of these: GPUNet-1, GPUNet-2, GPUNet-P0, GPUNet-P1, GPUNet-D1 or GPUNet-D2.
+### Install pre-requisites
+To run the example you need some extra python packages installed. These are needed for preprocessing images and visualization.
+```python
+!pip install validators matplotlib
+!pip install timm==0.5.4
+```
+
+```python
+import torch
+from PIL import Image
+import torchvision.transforms as transforms
+import numpy as np
+import json
+import requests
+import matplotlib.pyplot as plt
+import warnings
+
+warnings.filterwarnings('ignore')
+%matplotlib inline
+
+device = torch.device("cuda") 
+print(f'Using {device} for inference')
+
+if torch.cuda.is_available():
+    !nvidia-smi
+else:
+    torch.device("cpu")
+```
+
+### Load Pretrained model
+Loads NVIDIA GPUNet-0 model by default pre-trained on IMAGENET dataset. You can switch the default pre-trained model loading from GPUNet-0 to one of the following models listed below. 
+
+The model architecture is visible as output of the loaded model. For details architecture and latency info please refer to Figure#[6](https://drive.google.com/file/d/1pLA0OopYVRpYreIhQQKPdNauaaBEGITe/view) and Table#[3](https://drive.google.com/file/d/1pLA0OopYVRpYreIhQQKPdNauaaBEGITe/view) in the CVPR-2022 paper. 
```

Comment:
I personally don't have access to these figures

Reply:
Fixed